editor/editor_node.cpp

```diff
@@ -3356,6 +3384,14 @@ void EditorNode::_discard_changes(const String &p_str) {
 }
 
 void EditorNode::_update_file_menu_opened() {
+	bool has_unsaved = false;
+	for (int i = 0; i < editor_data.get_edited_scene_count(); i++) {
+		if (_is_scene_unsaved(i)) {
+			has_unsaved = true;
+			break;
+		}
+	}
+	file_menu->set_item_disabled(file_menu->get_item_index(FILE_SAVE_ALL_SCENES), !has_unsaved);
```

Comment:
Will it seem clear enough to the user why this is disabled?

Reply:
I can add a tooltip.

Reply:
Added a tooltip, although we don't really do that anywhere else.